### ISSUE TITLE
Optimize and cache image assets

### ISSUE THREAD

**Issue by s-kennedy** [CivicTechTO/women-and-color-backend#68]
We got 17/100 on the PageSpeed test, and the biggest issue is all the profile images we're loading:
https://developers.google.com/speed/pagespeed/insights/?url=beta.womenandcolor.com&tab=desktop

- [ ] Check the tips here: https://developers.google.com/web/fundamentals/design-and-ux/responsive/images
- [ ] This looks promising: https://github.com/sourcey/s3-image-optimizer
- [ ] About caching static files: http://staticfiles.productiondjango.com/blog/setting-up-static-file-caching/

Basically, we need to get the image sizes down and cache them with a really long expiry.

**Comment by s-kennedy** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Optimize image storage](https://trello.com/c/MLotz47l/60-optimize-image-storage)
